robots/open-manipulator/index.markdown

```diff
@@ -0,0 +1,45 @@
+---
+author: yoshimalucky
+comments: false
+date: 2018-10-26 15:10:00+00:00
+layout: page
+slug: open-manipulator
+title: Open Manipulator
+---
+
+
+	
+  * Company: Robotis
+
+	
+  * Website:
+
+	
+  * MoveIt! Config Package:
```

Comment:
can you add a link to the config package?

and the website, and your name as maintainer?